### DISCUSSION TITLE
Open sourcing the studio (with a MIT license)

### DISCUSSION THREAD

**Discussion by sgenoud** [sgenoud/replicad#100] [Announcements]
I have recieved mutliple requests on open sourcing the studio - that I kept so far closed. After multiple back and forth with myself I have decided that the studio should be part of the replicad project, and it is now in packages/studio!

Also, while I was making things more open, I have changed the license of the whole project to MIT.

**Comment by raydeleu** [comment]
This is so great! My greatest worry is always that a tool is abandoned after some time. Having it open source allows other people to take over the work and keep a project alive. If we consider Blender, this tool really took off after becoming open source, whilst all others are struggling. And you can still
make a good business out of it as soon as you have drawn sufficient users into actually using your tools. Just yesterday I was recreating an old 3D model in OnShape, which now also has parametric capabilities. OnShape works great, but I have earlier experienced a situation where I paid a large amount for 3D software that would not run 5 years later. I rather spend some additional effort in creating a model, knowing that the tool will still be available later. Blender is a good example. Models that I created shortly after Blender became open source can often still be opened in the current version. Even if its only for reference, being able to reuse old work is a large benefit. 

By the way, the recent addition of the editor to the workspace is fantastic. This works so much better than finding a separate editor, loading files into the workspace and then jumping back to the editor.
